File: api/src/main/java/br/cefet/aida/model/Atleta.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package br.cefet.aida.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 *
 * @author dougl
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Atleta {
    private int idAtleta;
    private String nome;
    private String email;
    private String cpf;
    private int idTurma;
    private Turma turma;
    private int idCurso;
    private Curso curso; 
         
}

File: site/src/app/model/atleta.ts
import { Curso } from "./curso";
import { Turma } from "./turma";

export class Atleta {
    idAtleta: number;
    nome: string;
    email: string;
    cpf: string;
    idTurma: number;
    turma: Turma;
    idCurso: number;
    curso: Curso;

    constructor() {
        this.idAtleta = 0;
        this.nome = "";
        this.email = "";
        this.cpf = "";
        this.idTurma = 0;
        this.turma = new Turma();
        this.idCurso = 0;
        this.curso = new Curso();
    }
}
